setup.sh

```diff
@@ -188,6 +188,46 @@ function arduino_setup() {
     arduino-cli core install arduino:avr
 }
 
+function setup_go() {
+    install_go
+
+    # Install tools for Go
+    # (currently none, `setec` didn't work out)
+}
+
+function install_go() {
```

Comment:
secrets management exploration is not going well, but this seemed useful enough to keep around